### ISSUE TITLE
pretrained weights

### ISSUE THREAD

**Issue by medicineqian** [heitorrapela/HalluciDet#2]
Hi, can you provide the test code and pretrained weights?

**Comment by heitorrapela** [comment]
Hi, thanks for your interest in our work. 
The training script in the main repo can also be used for test, but I will provide an easier way to do it in the next update. 
For the weights, I will also provide easy access to it, probably in the next few days I will share here,
Best Regards,
Heitor Medeiros

**Comment by heitorrapela** [comment]
Hi, sorry for the delay, we updated the code, now we have the [eval_hallucidet.py](https://github.com/heitorrapela/HalluciDet/blob/main/eval_hallucidet.py) script, and some pre-trained weights https://huggingface.co/heitorrapela/hallucidet/tree/main , to eval hallucidet, you have to download the detector and hallucidet weights.

After you download, you can place the weights inside ./checkpoints/ and run the following command:

`CUDA_VISIBLE_DEVICES=0 python eval_hallucidet.py --pretrained --modality ir --detector-path ./checkpoints/llvip/seed123/fasterrcnn_rgb_llvip_seed123.ckpt --hallucidet-path ./checkpoints/llvip/seed123/hallucidet_llvip_seed123.ckpt --wandb-project wacv2024 --wandb-name detector_fasterrcnn_hallucidet_det01reg01_llvip_200ep_seed123 --detector fasterrcnn --dataset llvip --epochs 1 --batch 8 --seed 123
`
